controllers/src/main/scala/cromwell/pipeline/controller/AkkaHttpClient.scala

```diff
@@ -45,11 +50,10 @@ class AkkaHttpClient(implicit actorSystem: ActorSystem, materializer: ActorMater
     params: Map[String, String] = Map(),
     headers: Map[String, String] = Map(),
     payload: String
-  )(implicit ec: ExecutionContext): Future[Response] = {
+  )(
+    implicit ec: ExecutionContext): Future[Response] = {
```

Comment:
What is this? Is this right way formatter does its work? ("newlines.implicitParamListModifier = [before]" ?) Why do we need this setting?

Reply:
Agree. In my opinion, we didn`t need this!

Reply:
ok
i've changed the behaviour